### DISCUSSION TITLE
Zksync

### DISCUSSION THREAD

**Discussion by vigneshk004** [Cyfrin/foundry-full-course-cu#2704] [Q&A]
When I am trying the plug in via zksync, the terminal is displaying the message that zksync is not maintained by remix and you are using an outdated version of the zksync plug in :v0.6.1-7 
Please update to v0.6.2-24 by force-refreshing the page and clearing your browser cache. I have tried refreshing but its still not working..

**Comment by EngrPips** [comment]
Did you try to update the plug-in as the error suggested?

**Reply by vigneshk004** [reply]
Yes

**Reply by EngrPips** [reply]
Honestly, working with `zksync` isn't currently smooth right now so you can skip the `zksync` part if it is causing you problem as it is optional.
